xgb_Wd_sftmx_md100_x14x2
{'lr': 0.0001, 'batch_size': 20000, 'N_epochs': 300, 'seed': 0}
Traceback (most recent call last):
  File "/vols/cms/hw423/Week14/A_Post_trains/InferenceAware_Trainning.py", line 193, in <module>
    oc = np.load(f'/vols/cms/hw423/Data/Week14/octest_{filecode}.npy')
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/vols/cms/hw423/env/envs/env/lib/python3.11/site-packages/numpy/lib/npyio.py", line 427, in load
    fid = stack.enter_context(open(os_fspath(file), "rb"))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/vols/cms/hw423/Data/Week14/octest_xgb_Wd_sftmx_md100.npy'
